pkg/gofr/handler_test.go

```diff
@@ -65,11 +65,12 @@ func TestHandler_ServeHTTP_Timeout(t *testing.T) {
 	w := httptest.NewRecorder()
 	r := httptest.NewRequest(http.MethodGet, "/", http.NoBody)
 
-	h := handler{requestTimeout: "1"}
+	// TODO: use a time.Duration for requestTimeout
+	h := handler{requestTimeout: "1"} // the minimum value is 1s
 
 	h.container = &container.Container{Logger: logging.NewLogger(logging.FATAL)}
 	h.function = func(*Context) (interface{}, error) {
-		time.Sleep(2 * time.Second)
+		time.Sleep(1200 * time.Millisecond)
```

Comment:
Here you can see the link between a delay that cannot be shorter than 1s and the 1200ms we have to wait

Reply:
i see this linking but didn't figured out how to reduce this sleep